components/browser/domains/build.gradle

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
We'll have to add the following block somewhere in here so that Android Studio stops complaining about coroutines being experimental:

```Groovy
kotlin {
    experimental {
         coroutines 'enable' 
    }
}
```

Reply:
Done.